draft-ietf-trans-rfc6962-bis.md

```diff
@@ -312,57 +312,57 @@ list of missing nodes required to compute the nodes leading from a leaf to the
 root of the tree. If the root computed from the inclusion proof matches the true
 root, then the inclusion proof proves that the leaf exists in the tree.
 
-Given an ordered list of n inputs to the tree, D\[n] = {d(0), ..., d(n-1)}, the
-Merkle inclusion proof PATH(m, D\[n]) for the (m+1)th input d(m), 0 \<= m \< n,
-is defined as follows:
+Given an ordered list of n inputs to the tree, D^(n) = {d[0], d[1], ...,
+d[n-1]}, the Merkle inclusion proof PATH(m, D^(n)) for the (m+1)th input d[m], 0
+\<= m \< n, is defined as follows:
 
 The proof for the single leaf in a tree with a one-element input list D\[1] =
-{d(0)} is empty:
+{d[0]} is empty:
 
 ~~~~~~~~~~~
-PATH(0, {d(0)}) = {}
+PATH(0, {d[0]}) = {}
 ~~~~~~~~~~~
 
 For n > 1, let k be the largest power of two smaller than n. The proof for the
-(m+1)th element d(m) in a list of n > m elements is then defined recursively as
+(m+1)th element d[m] in a list of n > m elements is then defined recursively as
 
 ~~~~~~~~~~~
-PATH(m, D[n]) = PATH(m, D[0:k]) : MTH(D[k:n]) for m < k; and
+PATH(m, D^(n)) = PATH(m, D[0:k]) : MTH(D[k:n]) for m < k; and
 
-PATH(m, D[n]) = PATH(m - k, D[k:n]) : MTH(D[0:k]) for m >= k,
+PATH(m, D^(n)) = PATH(m - k, D[k:n]) : MTH(D[0:k]) for m >= k,
 ~~~~~~~~~~~
 
-where : is concatenation of lists and D\[k1:k2] denotes the length (k2 - k1)
-list {d(k1), d(k1+1),..., d(k2-1)} as before.
+Here, : is a concatenation of lists, and as before, D\[k1:k2] denotes the list
+{d[k1], d[k1+1], ..., d[k2-1]} of length (k2 - k1).
```

Comment:
This is the core problem ... D[k1:k2] is really a new tree, so having different notation and numbering than the original tree is confusing. It occurs to me that we could say something like:

D[k1:k2] = D'_{k2 - k1} denotes the list {d'[0] = d[k1], d'[1] = d[k1 + 1], ... , d'[k2 - k1 - 1] = d[k2 - 1]}

and that fixes the weirdness of the list numbering...